canonical-web-and-design/maas-ui
# Fix processing state

## Done
- Prevent the "update your selection" notice from appearing once machines have begun to be processed. This was causing a bug as when the machines begin to be changed they can change the allowable actions from that machine, resulting in that machine being shown.

## QA
- Selecting a number of new machines (at least 5).
- Using the take action menu select "Commission" and submit the form.
- The machines should transition to commissioning and the "update your selection" warning should not appear.
- You might have to do this a few times (you can abort once they transition to commissioning) to be satisfied this is fixed.

## Fixes
Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1897.
